CODEOWNERS

```diff
@@ -37,24 +37,24 @@ modules/bitgo/test/v2/unit/coins/staking/ @BitGo/eth-team
 modules/bitgo/test/v2/unit/coins/token.ts @BitGo/eth-team
 
 # All other coins
-modules/sdk-coin-aca/ @BitGo/coins
-modules/sdk-coin-ada/ @BitGo/coins
-modules/sdk-coin-algo/ @BitGo/coins
-modules/sdk-coin-avaxc/ @BitGo/coins
-modules/sdk-coin-avaxp/ @BitGo/coins
-modules/sdk-coin-bsc/ @BitGo/coins
-modules/sdk-coin-cspr/ @BitGo/coins
-modules/sdk-coin-dot/ @BitGo/coins
-modules/sdk-coin-eos/ @BitGo/coins
-modules/sdk-coin-hbar/ @BitGo/coins
-modules/sdk-coin-near/ @BitGo/coins
+modules/sdk-coin-aca/ @BitGo/blockchain-integration
+modules/sdk-coin-ada/ @BitGo/blockchain-integration
+modules/sdk-coin-algo/ @BitGo/blockchain-integration
+modules/sdk-coin-avaxc/ @BitGo/blockchain-integration
+modules/sdk-coin-avaxp/ @BitGo/blockchain-integration
+modules/sdk-coin-bsc/ @BitGo/eth-team
+modules/sdk-coin-cspr/ @BitGo/blockchain-integration
+modules/sdk-coin-dot/ @BitGo/blockchain-integration
+modules/sdk-coin-eos/ @BitGo/blockchain-integration
+modules/sdk-coin-hbar/ @BitGo/blockchain-integration
+modules/sdk-coin-near/ @BitGo/blockchain-integration
 modules/sdk-coin-polygon/ @BitGo/coins
```

Comment:
Since we are at it, is there a team for polygon or this should go to the eth-team?